tests/system/small/test_dataframe_io.py

```diff
@@ -150,7 +149,7 @@ def test_to_csv_index(
     # read_csv will decode into bytes inproperly, convert_pandas_dtypes will encode properly from string
     dtype.pop("bytes_col")
     gcs_df = pd.read_csv(
-        path,
+        path.replace("*", "000000000000"),
```

Comment:
might make sense to move `FIRST_FILE = "000000000000"` from test_session.py to utils.py and reuse everywhere

Reply:
Good idea. Addressed.